packages/react-components/src/LinkExternal.tsx

```diff
@@ -52,7 +52,7 @@ function genLinks (systemChain: string, { data, hash, isLogo, isSidebar, type }:
                 src={logo}
               />
             )
-            : name
+            : 'Show in ' + name
```

Comment:
This should be translatable. Additionally, should be reverted - if you look at the explorer view for a block, it is all-over the page now.

```suggestion
            : name
```